include/core/parameters_lagrangian.h

```diff
@@ -178,33 +179,41 @@ namespace Parameters
       // Insertion frequency
       int insertion_frequency;
 
-      // Direction (axis) of insertion of particles (1st, 2nd, 3rd)
-      std::vector<unsigned int> direction_sequence;
+      //// Clear and Add :
```

Comment:
```suggestion
      // Clear and Add:
```

Reply:
I will do like Bruno suggested with the /* */